filtered-resource-service/src/main/java/uk/gov/gchq/palisade/service/filteredresource/web/router/KafkaRestWriterRouter.java

```diff
@@ -36,6 +36,9 @@
 import java.util.stream.Collectors;
```

Comment:
SonarQube smell relating to the use of 200, wants you to assign it to a constant and use that instead